src/main/java/net/glowstone/EventFactory.java

```diff
@@ -49,15 +52,16 @@ private EventFactory() {
      * @return the called event
      */
     public static <T extends Event> T callEvent(T event) {
-        GlowServer server = (GlowServer) Bukkit.getServer();
+        Server server = Bukkit.getServer();
 
```

Comment:
Why was this change necessary? `Bukkit.getServer()` is guarenteed to be a `GlowServer` instance. The only difference I see here are the additional imports

Reply:
Yes it did (ServerShim doesn't extend GlowServer).